ios/Classes/Blemulator/Model/ConnectionStateEvent.m

```diff
@@ -0,0 +1,21 @@
+#import "ConnectionStateEvent.h"
+
+@implementation ConnectionStateEvent
+
+- (instancetype)initWith:(NSString *)deviceId connectionState:(NSString *)connectionState {
+    self = [super init];
+    if (self) {
+        self.deviceId = deviceId;
+        self.connectionState = connectionState;
+    }
+    return self;
+}
+
+- (NSDictionary<NSString *,id> *)jsonObjectRepresentation {
+    return [NSDictionary dictionaryWithObjectsAndKeys:
+            _deviceId, @"id",
```

Comment:
Will those strings be reused? Perhaps they should be extracted?

Reply:
I'm not sure yet if they are going to be reused. I've extracted them to local constants for better code structure though - it's possible to make them public if there's a need in the future.